src/common/inc/block_builder.h

```diff
@@ -20,6 +20,8 @@
 #define BLOCK_BUILDER_CSV_FIELD_STATION_SIDE         7
 #define BLOCK_BUILDER_CSV_FIELD_ELEVATION            8
 #define BLOCK_BUILDER_CSV_FIELD_CUMULATIVE_ELEVATION 9
+#define BLOCK_BUILDER_CSV_FIELD_CONNECTION           11
```

Comment:
What is field 10?

Reply:
field 10 is the seconds to traverse block field in both the csv's